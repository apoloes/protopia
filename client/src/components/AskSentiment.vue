<template>
  <!--  <div class="content-header h4 text-center pt-2 pb-3">Highlights</div>-->
  <div class="custom-card header-card card">
    <!--    <div class="card-body pb-0">-->
    <div class="content-header h4 text-center pt-2 pb-3">Highlights</div>
    <pos-neg :cleanAskData="cleanAskData"></pos-neg>
    <top-pos :cleanAskData="cleanAskData"></top-pos>
    <top-neg :cleanAskData="cleanAskData"></top-neg>
    <!--    </div>-->
  </div>
</template>

<script>
import PosNeg from "./PosNeg.vue";
import TopPos from "./TopPos.vue";
import TopNeg from "./TopNeg.vue";
export default {
  props: ["cleanAskData"],
  components: {
    "pos-neg": PosNeg,
    "top-pos": TopPos,
    "top-neg": TopNeg,
  },
};
</script>
