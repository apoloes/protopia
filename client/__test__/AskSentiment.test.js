import { shallowMount } from '@vue/test-utils'
import AskSentiment from '../src/components/AskSentiment'

describe('AskSentiment.vue', () => {
  it('renders item in props.items', () => {
    const wrapper = shallowMount(AskSentiment);
    expect(wrapper.vm.$options._propKeys).toHaveLength(1)
  })


  it('matches snapshot', () => {
    const wrapper = shallowMount(AskSentiment);
    expect(wrapper.html()).toMatchSnapshot();
  })
})
