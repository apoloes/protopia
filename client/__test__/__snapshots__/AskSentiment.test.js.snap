// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AskSentiment.vue matches snapshot 1`] = `
<div class="custom-content-card content-card card">
  <div class="card-body pb-0">
    <div class="content-header h4 text-center pt-2 pb-3">Highlights</div>
    <pos-neg-stub></pos-neg-stub>
    <top-pos-stub></top-pos-stub>
    <top-neg-stub></top-neg-stub>
  </div>
</div>
`;
